app/Http/Controllers/MessagesController.php

```diff
@@ -92,16 +92,20 @@ public function sendMessage(Message $message)
     {
         // Get competition with activity.
         $competitionId = request('competition_id');
-        $competition = Competition::find($competitionId);
+        $competition = null;
```

Comment:
Also this is a test, so it shouldn't even get there. Since you are already checking if the competition ID was sent in the request, I think you can just check if the competition exists in the when setting that property in the `resources` array.

```
        $resources = [
            'message' => $message,
            'competition' => isset($competition) ? $competition : null,
            'users' => $users,
            'test' => request('test'),
        ];
```